<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="lEngine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngine">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPiecesStyle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lPiecesStyle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Pieces style</string>
   </property>
  </widget>
  <widget class="QLabel" name="lGameControl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Game control</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPlayerTime">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Player time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lEngineTime">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine time</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPlayerTime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngineTime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="lOtherSettings">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Other settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="lGeneralSettings">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string notr="true">General settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPlayerName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Player name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edPlayerName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lInterfaceLanguage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>627</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Interface language</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbInterfaceLanguage">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>629</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="lEngineProfile">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine profile</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngineProfile">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDrawCoordinates">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>478</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Draw coordinates on board sides</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDrawMoveArrow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>445</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Draw last move arrow</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAutoSaveGames">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>578</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Auto save games</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbQuickSingleMoveSelection">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>548</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Allow quick selection of single moves</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbShowMoveHints">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>511</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show move hints</string>
   </property>
  </widget>
  <widget class="QLabel" name="lBoardMargins">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Board margins</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoardMargins">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>399</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>4</number>
   </property>
   <property name="value">
    <number>28</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>710</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edPlayerName</tabstop>
  <tabstop>cmbEngine</tabstop>
  <tabstop>cmbEngineProfile</tabstop>
  <tabstop>cmbPiecesStyle</tabstop>
  <tabstop>cmbPlayerTime</tabstop>
  <tabstop>cmbEngineTime</tabstop>
  <tabstop>spinBoardMargins</tabstop>
  <tabstop>cbDrawMoveArrow</tabstop>
  <tabstop>cbDrawCoordinates</tabstop>
  <tabstop>cbShowMoveHints</tabstop>
  <tabstop>cbQuickSingleMoveSelection</tabstop>
  <tabstop>cbAutoSaveGames</tabstop>
  <tabstop>cmbInterfaceLanguage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
